configure.ac

```diff
@@ -453,10 +453,11 @@ DX_XML_FEATURE(OFF)
 DX_PDF_FEATURE(OFF)
 DX_PS_FEATURE(OFF)
 DX_INIT_DOXYGEN($PACKAGE_NAME, [Doxyfile], [doxygen-doc])
-AM_CONDITIONAL(DOXYMAN, [test $DX_FLAG_man -eq 1])
 
-AS_IF([test "x$enable_doxygen_doc" != xno],
-      [ERROR_IF_NO_PROG([doxygen])])
+AM_CONDITIONAL(DOXYMAN, [test $DX_FLAG_man -eq 1])
+AS_IF([test $DX_FLAG_man -ne 1],
```

Comment:
Perhaps we should use the format independent flag $DX_FLAG_doc?

Reply:
Thank you very much for the suggestion you're right, i make the changes and they work correctly

Reply:
What about $DX_FLAG_man in line 458?

Reply:
A little mistake, now its already changed. Thanks